### PR TITLE
Add project structure for basic music reproduction / Missing key from spotify

### PR DESCRIPTION
**Pull request by the-guti**
Fix readme look / Add project structure for front and back / Add basic html for music reproduction / Missing Key / Add pom and app-engine with id to be able to deploy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.google.sps</groupId>
```
Comment (chrisachille) [inline pom.xml:6]: Would it make sense to change this or the artifactId?  If not known, we can easily do it later.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <artifactId>appengine-maven-plugin</artifactId>
+            <version>2.2.0</version>
+            <configuration>
+                <deploy.projectId>summer20-sps-6</deploy.projectId>
+                <deploy.version>1</deploy.version>
+            </configuration>
+        </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (chrisachille) [inline pom.xml:49]: nit: Add extra carriage-return.

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (chrisachille) [inline src/main/webapp/WEB-INF/appengine-web.xml:1]: Nit: If sticking with 4 spacing, then format this file to conform as well.

**Review by chrisachille — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (the-guti) [reply pom.xml:6]: We need to check what it does. I really dont know

File: src/main/webapp/WEB-INF/appengine-web.xml (hunk 3 above)
Comment (the-guti) [reply src/main/webapp/WEB-INF/appengine-web.xml:1]: Sure!

File: pom.xml (hunk 2 above)
Comment (the-guti) [reply pom.xml:49]: Done

**Pull request merged**